atmo/templates/atmo/base.html

```diff
@@ -97,6 +97,7 @@
     <script type="text/javascript" src="{% static "npm/bootstrap-datetime-picker/js/bootstrap-datetimepicker.min.js" %}"></script>
     <script type="text/javascript" src="{% static "npm/parsleyjs/dist/parsley.min.js" %}"></script>
     <script type="text/javascript" src="{% static "js/base.js" %}"></script>
+    <script type="text/javascript" src="{% static "js/fileinput.js" %}"></script>
```

Comment:
This was kind of a huge omission when I moved the jasny stuff in f47ab9b3f735efd7925962d2d22f02bcaccddab9. My bad :(